### PR TITLE
Radio background

### PR DESCRIPTION
**Pull request by rafaelab**
Add models of the radio background (Nitu21, Fixsen11).

### CODE REVIEW

File: photonField.py [hunk 1]
```diff
@@ -276,25 +277,95 @@ def getEmax(self, z=0):
...
+
+        I[eps < self.getEmin()] = 0.
+        I[eps > self.getEmax()] = 0.
+
+        return I
 
     def getEmin(self, z=0):
         """Minimum effective photon energy in [J]"""
-        return 1e-10 * eV
+        return 1e3 * h
```
Comment (lukasmerten) [inline photonField.py:360]: Is this really correct; units of h = m^2 * kg / s=J*s != J ?

File: photonField.py [hunk 2]
```diff
@@ -276,25 +277,95 @@ def getEmax(self, z=0):
...
 
     def getEmax(self, z=0):
         """Maximum effective photon energy in [J]"""
-        return 0.1 * eV
-
+        return 1e12 * h
```
Comment (lukasmerten) [inline photonField.py:364]: Is this really correct; units of h = m^2 * kg / s=J*s != J ?

File: photonField.py [hunk 3]
```diff
@@ -276,25 +277,95 @@ def getEmax(self, z=0):
...
+        I[eps > self.getEmax()] = 0.
+        return I
+
+    def getEmin(self, z = 0.):
+        """Minimum effective photon energy in [J]"""
+        return 2.2e6 * h
+
+    def getEmax(self, z = 0.):
+        """Maximum effective photon energy in [J]"""
+        return 1e10 * h
```
Comment (lukasmerten) [inline photonField.py:313]: Is this really correct; units of h = m^2 * kg / s=J*s != J ?

File: photonField.py [hunk 4]
```diff
@@ -276,25 +277,95 @@ def getEmax(self, z=0):
...
+        nu = eps / h
+        T = T_CMB + 24.1 * np.power(nu / 3.1e8, -2.6)
+        I = 8. * np.pi / c0 ** 3 / h ** 3 * eps ** 2 / (np.exp(eps / (kB * T)) - 1.)
+        I[eps < self.getEmin()] = 0.
+        I[eps > self.getEmax()] = 0.
+        return I
+
+    def getEmin(self, z = 0.):
+        """Minimum effective photon energy in [J]"""
+        return 2.2e6 * h
```
Comment (lukasmerten) [inline photonField.py:309]: Is this really correct; units of h = m^2 * kg / s=J*s != J ?

**Comment by lukasmerten** [comment]
Apart from the min/max energies it looks good to me.

File: photonField.py (hunk 4 above)
Comment (rafaelab) [reply photonField.py:309]: 2.2e6 Hz is the frequency, which goes up to ~10 GHz (figure 5 of the paper). Therefore, the units are correct.

File: photonField.py (hunk 3 above)
Comment (rafaelab) [reply photonField.py:313]: 1e10 Hz is the frequency.

File: photonField.py (hunk 1 above)
Comment (rafaelab) [reply photonField.py:360]: This is also correct. This model is valid in the range between 1e3 Hz and 1e12 Hz (see figure 3 of the corresponding paper).

Comment (lukasmerten) [reply photonField.py:364]: I see. Maybe we could write 1e3 * 1/second * h or define Hz in Units.h and write 1e3 * Hz * h? Same applies for the other occasions.

Comment (rafaelab) [reply photonField.py]: > I see. Maybe we could write 1e3 * 1/second * h or define Hz in Units.h and write 1e3 * Hz * h? Same applies for the other occasions.

I defined Hz in the beginning of photonField.py.

**Pull request merged**